packages/block-editor/src/components/inserter/hooks/use-block-types-state.js

```diff
@@ -26,8 +26,8 @@ const createBlocksFromInnerBlocksTemplate = ( innerBlocksTemplate ) => {
 /**
  * Retrieves the block types inserter state.
  *
- * @param {string=}  rootClientId        Insertion's root client ID.
- * @param {Function} onInsert            function called when inserter a list of blocks.
+ * @param {string=}  rootClientId Insertion's root client ID.
+ * @param {Function} onInsert     function called when inserter a list of blocks.
```

Comment:
```suggestion
 * @param {Function} onInsert     Function called when inserter a list of blocks.
```

Should descriptions always begin with a Capital letter? It appears to be the convention and is also a PHP coding standard